src/geovista/examples/extraction/wedge_manifold.py

```diff
@@ -8,21 +8,36 @@
 Wedge Extraction
 ----------------
 
-This example demonstrates how to extract a region from a mesh using a geodesic manifold.
+This example demonstrates how to extract a region using a geodesic wedge manifold.
 
 📋 Summary
 ^^^^^^^^^^
 Creates a mesh from 1-D latitude and longitude unstructured points and
 connectivity.
 
-Isolates wedges of this mesh and plots them onto natural earth base layer
-texture.
-
 It uses an unstructured Met Office LFRic C48 cubed-sphere of surface altitude
 data.
 
-The resulting mesh contains quad cells and is constructed from CF UGRID
-unstructured cell points and connectivity.
+Three separate geodesic wedge manifolds are constructed to extract the cells of the
+mesh contained within the wedge. Each wedge uses a different enclosure preference to
+extract the cells.
+
+The regions extracted by each wedge are rendered along with the boundary where the
+manifold intersects the surface of the mesh. A different colour is used for each
+manifold boundary.
+
+The **red boundary** contains only those cells where all points defining the face of a
+cell are within the manifold.
+
+The **purple boundary** contains only those cells where at least 1 point that defines
+the face of the cell is within the manifold.
+
+The **orange boundary** contains only those cells where the face cell center is within
```

Comment:
```suggestion
The **orange boundary** contains only those cells where the center of the cell is within
```

Alternatively, `where the center of the face of the cell...`, though that's a bit clunky. If "face cell center" is nomenclature, ignore this!